src/Game.js

```diff
@@ -110,8 +113,8 @@ class Game
 			includeCaptureEligible = true;
 		}
 
-		validity = vectorList.some((vector) => {
-			this.board.GetCellIndices(vector, move.source, includeCaptureEligible).includes(move.target);
+		vectorList.forEach((vector) => {
```

Comment:
What was the reason for this? Performance? Is `GetCellIndices` an expensive operation?

Reply:
It fixed a bug. I know the two should be equivalent, but they aren't. Validity was coming back false, even though I confirmed by hand that there was at least one vector where the function returned true, and that the vector was being tested in some. Changing to this fixed the issue. The new version is also mildly more efficient because short-circuiting, which is a nice plus.